### PR TITLE
Update button label to 'See available versions' in docs homepage

### PR DESCRIPTION
**Pull request by steve-hardman**
Changed link of "About the documentation" to button with text "See available versions" on homepage

This will fix https://github.com/kubernetes/website/issues/43249

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle Update button label to 'See available versions' in docs homepage

File: content/en/docs/home/_index.md [hunk 1]
```diff
@@ -64,4 +64,6 @@ cards:
 - name: about
   title: About the documentation
   description: This website contains documentation for the current and previous 4 versions of Kubernetes.
+  button: "See available versions"
+  button_path: "docs/supported-doc-versions"
```
Comment (dipesh-rawat) [inline content/en/docs/home/_index.md:68]: The button link is [currently](https://deploy-preview-45571--kubernetes-io-main-staging.netlify.app/docs/home/) directing to a non-existent path. Please consider the following suggestion.

```suggestion
  button_path: "/docs/home/supported-doc-versions"
```

**Comment by dipesh-rawat** [comment]
@steve-hardman If this pull request aims to address an existing issue, it would be beneficial to include this information in the [PR description](https://github.com/kubernetes/website/pull/45571#issue-2190252530). This addition will provide reviewers with context, aiding in their understanding of the changes being proposed.

File: content/en/docs/home/_index.md (hunk 1 above)
Comment (steve-hardman) [reply content/en/docs/home/_index.md]: Thanks for review. I have completed all the changes as requested.

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!
/lgtm

[Preview Page](https://deploy-preview-45571--kubernetes-io-main-staging.netlify.app/docs/home/)

**Comment by steve-hardman** [comment]
> [APPROVALNOTIFIER] This PR is NOT APPROVED
This pull-request has been approved by:
Once this PR has been reviewed and has the lgtm label, please assign [sftim](https://github.com/sftim) for approval. For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

Trying to do bot's suggestion. (sorry if it not right thing to do)

/assign @sftim

**Pull request merged**